.vscode/settings.json

```diff
@@ -30,5 +30,17 @@
   "typescript.enablePromptUseWorkspaceTsdk": true,
   "typescript.format.enable": false,
   "typescript.preferences.quoteStyle": "double",
-  "typescript.tsdk": "node_modules/typescript/lib"
-}
\ No newline at end of file
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "cSpell.words": [
+    "aspnet",
+    "Beachball",
+    "Blazor",
+    "combobox",
+    "fluentui",
```

Comment:
Are these just here to prevent errors showing up in vscode? This is the package name (technically `@fluentui`). FWIW, if this only exists to suppress something showing up in vscode as a spelling error, I'd prefer we pull this.

Reply:
This essentially ads the names as dictionary items for the environment. They will always show up as squiggly lines if not added.  This means the developer experience takes a hit every time you acknowledge one of these words to be ignored other than being an actual problem that requires your attention. So I disagree.  I don't think there is any problem with adding this and in fact, it's becoming more common.

Reply:
Eh, I personally think it's a stretch to say that this impacts dev ex. It depends on having the extension installed and using it. I think my big concern here is, "What is the line?"

How long do we let this list get as our documentation grows and we list out additional packages, code examples (assume that's where `typeof` and `wwwroot` are coming from). At a certain point, it seems more like the issue is with the tool than the actual content. If it can't tell the difference between a code snippet in markdown and text, this is going to get out of hand everytime we want to reference something, etc. So, do we have a line, or is our expectation that this file grows any time we see red squigglies? That's my biggest question with starting down this road.

Reply:
The file length has zero impact.  It's not compiled or configured and this feature doesn't require any extension to be installed.  These files will expand over time, especially as more IaC is implemented.